python/mlc_chat/help.py

```diff
@@ -101,6 +101,10 @@
     "attention_sink_size": """
 (Experimental) The number of stored sinks. Only supported on Mistral yet. By default,
 the number of sinks is 4. This flag subjects to future refactoring.
+""".strip(),
+    "max_batch_size": """
+The maximum allowed batch size set for batch prefill/decode function.
+By default, the maximum batch size is 80.
```

Comment:
No need to mention its default here, because argparse will generate its value if `"%(default)s"` is used